process/subgroups.md

```diff
@@ -0,0 +1,66 @@
+# Subgroups
+
+Subgroups of the CG are created when one (or both) of the following apply:
```

Comment:
It may also be useful to add a line about the mechanics of creating a subgroup, i.e. poll for creating the relevant subgroup, and provide information about who will be chairing that subgroup at a CG meeting.

Reply:
Good idea. I added the following wording:

> To create a subgroup, you should add a poll to a CG meeting agenda with at least 7 days notice. As part of the poll, you should provide information about who will be chairing the subgroup.